### ISSUE TITLE
Plugin update indicates an error when it should be a warning

### ISSUE THREAD

**Issue by Starker3** [piwik/piwik#19284]
With some recent plugin updates there were higher minimum versions set for those plugins.
With users who have an install of Matomo that is not the latest version, the plugins page shows that plugin updates are available but show a big red warning that the plugin isn't compatible with their version of Matomo. Eg.
![image](https://user-images.githubusercontent.com/20905821/170892884-62f16ef6-7e86-428c-b650-63c4b405122c.png)

For Many users this can cause them to think that the version of the plugin they currently have installed is not compatible. We could perhaps show a different message to indicate that the version they are trying to update to is not compatible with their version of Matomo and perhaps change it to a warning (Eg. Orange/Yellow) instead of an error (Red). For example:
![image](https://user-images.githubusercontent.com/20905821/170892989-4373fd66-3b96-4ad2-8273-ae2748f7f65f.png)

**Comment by peterhashair** [comment]
@Starker3 thanks for reporting this, our product team will process this ASAP

**Comment by AltamashShaikh** [comment]
@Starker3 The higher min version is due to recent migration to vue.js and it is expected

**Comment by Starker3** [comment]
@AltamashShaikh Yes, the min version being higher isn't the problem here. The problem here is that the warning isn't very clear and can cause confusion.
